components/business/reservations/reservation-record-index.tsx

```diff
@@ -33,9 +33,10 @@ export default async function ReservationRecordIndex({
                                                          accessCode,
                                                          status
                                                      }: ReservationRecordIndexProps) {
-    return <tr className={"hover:bg-muted shrink-0 truncate"}>
+    return <tr
+        className={"hover:bg-muted shrink-0 truncate max-sm:[&>*:nth-child(3)]:hidden max-sm:[&>*:nth-child(4)]:hidden max-sm:[&>*:nth-child(5)]:hidden max-lg:[&>*:nth-child(6)]:hidden max-sm:[&>*:nth-child(7)]:hidden"}>
         <ReservationRecordDatapoint>{reservationYear.substring(0, 4) + '-' + reservationNumber}</ReservationRecordDatapoint>
-        <ReservationRecordDatapoint>{start_date === end_date ? start_date : start_date + " tot " + end_date}</ReservationRecordDatapoint>
+        <ReservationRecordDatapoint>{start_date === end_date ? start_date : start_date + " tot " + end_date.substring(5, 10)}</ReservationRecordDatapoint>
```

Comment:
```suggestion
        <ReservationRecordDatapoint>{start_date === end_date ? start_date : start_date + " tot " + end_date.toLocaleDateString()}</ReservationRecordDatapoint>
```
Ik weet de exacte naam nog steeds niet.
Maar hier bedoel ik hetzelfde als de vorige comment.